base/show.jl

```diff
@@ -469,6 +469,14 @@ end
 isidentifier(s::Symbol) = isidentifier(string(s))
 
 isoperator(s::Symbol) = ccall(:jl_is_operator, Cint, (Cstring,), s) != 0
+
+"""
+    operator_precedence(s::Symbol)
+
+Returns an integer representing the precedence of a Julia operator, relative
+to other operators. Higher numbers take precedence over lower-numbered
+operators. Returns `0` if a symbol is not a valid operator.
+"""
```

Comment:
Perhaps "a Julia operator" -> "operator `s`"?

Similarly, perhaps "if a symbol is not a valid operator" -> "if `s` is not a valid operator"?

For consistency, perhaps "Higher-numbered operators take precedence over lower-numbered operators."?